index.html

```diff
@@ -29,13 +37,14 @@ <h3 class="head3" id="social-1">LET’S CONNECT</h3>
               <li class="social-items"><a href="https://medium.com/"><img src="images/medium.png" alt="medium link"></a></li>
```

Comment:
- [ ] Please here is the right block in "About Myself section":
- Kindly make sure the Languages are displayed in a horizontal line.
- You design
![F1](https://user-images.githubusercontent.com/63972670/166111101-16b91a00-5083-4759-91ea-73eb93300439.PNG)
- Figma design

![f2](https://user-images.githubusercontent.com/63972670/166111119-0a8ea7d2-7ae7-4215-8a47-ff8a683fbaab.PNG)